### ISSUE TITLE
Audio Noise  - Horus

### ISSUE THREAD

**Issue by robthomson** [opentx/opentx#4137]
When using opentx on the horus - there is a persistant background hiss

This is not present when running the frsky os.

Possibile solution... turn the amp off when the radio is not playing sound?

**Comment by projectkk2glider** [comment]
Turning off the amplifier might even save some power, but two things to check:
 * verify that amp on/off does not produce noise
 * determine how quickly amp turns on, ie is it possible to turn it on when we have something to play and then send ADC data immediately?

**Comment by markran** [comment]
I would greatly appreciate this as the hiss from my $400 radio is a bit annoying (I fly in quiet places :-))

**Comment by Xerravon** [comment]
I get this same problem but with the crossfire TX installed. it is very loud when crossfire gets power. it is very loud when set to 868mhz but quieter when set to 915mhz or when the unit connects with the rx. It is very quite once the rx connects.
I also noticed if I set the crossfire from "normal" to "Forced" in telem setting it changes from a constant buzz sound to a pulsed sound...
hope this helps. I also sent TBS the same info.
thanks
John
